src/auth/screens/login.screen.js

```diff
@@ -289,10 +295,13 @@ class Login extends Component {
                 type="octicon"
               />
               <Text style={styles.title}>
-                {translate('auth.login.notificationsTitle', locale)}
+                <Trans>Control notifications</Trans>
```

Comment:
`<Trans>` can be used in JSX, just like you would use `<Text>`:

```
<View>
  <Trans>This will be translated</Trans>
</View>
```

When you need to translate a component prop however, you can't use that as `<Button title={<Trans>foo</Trans>} />` wouldn't make much sense as title expects a string, not a component.

This is when i18n.t comes into play: ```<Button title={i18n.t`foo`} />```